website/docs/r/deploy_key_enable.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "gitlab"
+page_title: "GitLab: gitlab_deploy_key_enable"
+sidebar_current: "docs-gitlab-resource-deploy_key_enable"
+description: |-
+  Enable a pre-existing deploy key in the project
+---
+
+# gitlab\_deploy\_key\_enable
+
+This resource allows you to enable pre-existing deploy keys for your GitLab projects.
+
+**the GITLAB KEY_ID for the deploy key must be known**
+
+## Example Usage
+
+```hcl
+resource "gitlab_deploy_key_enable" "example" {
+  project = "12345"
```

Comment:
please provide an example where we use variables. (do not hesitate to add a small gitlab_project to the example)

Reply:
will do